src/main/resources/ath-container/Dockerfile

```diff
@@ -33,14 +33,15 @@ RUN apt-get update && \
 
 # Install a fixed firefox version that is known to work with the current selenium version, copied from https://hub.docker.com/r/selenium/node-firefox/dockerfile
 ARG FIREFOX_VERSION=108.0.2
+# hadolint ignore=DL3008, DL3015
```

Comment:
About `DL3018`: do you mind adding a comment before the ignore rule to explain why you would want to ignore it? It helps to share knowledge of the "why".

Reply:
Do you mean 3008? We ignore it here the same way we ignore it above.

Reply:
my bad, I meant `DL3008ù` yes 👍 
No problem to not pin dependencies. But I really like adding a comment explaining "why" along with the inline ignore ;)

Reply:
for sure, added 👍🏻